dev/integration_tests/flutter_gallery/test_driver/transitions_perf_e2e.dart

```diff
@@ -0,0 +1,127 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
If we plan to use this file both in e2e and flutter_driver-timeline tests, maybe we should rename this file?

Reply:
Nope, this one is not reused. For the reused version `run_demos` function will be in a separate file.